optaplanner-examples/src/main/java/org/optaplanner/examples/coachshuttlegathering/domain/location/RoadLocationArc.java

```diff
@@ -67,4 +67,12 @@ public void setShuttleDuration(int shuttleDuration) {
     // Complex methods
     // ************************************************************************
 
+    @Override
+    public String toString() {
+        return "Road arc [coach: "
```

Comment:
Looking at Period.toString() and Machine.toString() I notice there is no consistency really currently (there should be though). I am not a fan of toStrings() that mention all their properties - they are the main cause of unreadble (too verbose) logging. But I agree in this case we don't have much choice.

I am ok with merging this as is once jenkins is green. I 'll go through all toStrings() at some point.